src/openklant/components/klantinteracties/api/tests/test_partijen.py

```diff
@@ -2142,6 +2143,49 @@ def _get_detail_url(partij: Partij) -> str:
             received_adressen[0]["url"], f"http://testserver{expected_url}"
         )
 
+    def test_voorkeurs_digitaal_adres_valid(self):
+        partij = PartijFactory.create()
+        digitaal_adres = DigitaalAdresFactory.create(partij=partij)
+        partij.voorkeurs_digitaal_adres = digitaal_adres
+
+        partij.clean()
+
+    def test_voorkeurs_digitaal_adres_invalid(self):
+        partij = PartijFactory.create()
+        invalid_digitaal_adres = DigitaalAdresFactory.create()
+        partij.voorkeurs_digitaal_adres = invalid_digitaal_adres
+
+        with self.assertRaises(ValidationError) as cm:
+            partij.clean()
+
+        self.assertIn(
+            "Het voorkeurs adres moet een gelinkte digitaal adres zijn.",
+            str(cm.exception),
+        )
```

Comment:
Just a minor suggest
```suggestion
self.assertEqual("Het voorkeurs adres moet een gelinkte digitaal adres zijn.", str(cm.exception.message))
```